.local/Dockerfile

```diff
@@ -5,16 +5,17 @@ ENV GOROOT=/usr/local/go
 ENV GOPATH=${HOME}/go
 ENV PATH=$PATH:${GOROOT}/bin
 
-RUN apk update && apk add --no-cache \
-    git && \
-    go get github.com/go-delve/delve/cmd/dlv
+RUN apk update && apk add --no-cache
+
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
 
 WORKDIR /secrets-store-csi-driver-codebase
 
-COPY go.mod go.mod
+COPY go.mod go.sum ./
 RUN go mod download
+RUN go mod tidy
```

Comment:
Why is `go mod tidy` required when we're running `go mod download` in the previous step?

Reply:
Add this as a comment in the the `Dockerfile`